.github/workflows/cleanup-resources-daily.yml

```diff
@@ -0,0 +1,44 @@
+# This exists because some actions might be creating resources AFTER the branch cleanup happened like with the previews.
+# As a result, we need to check for dangling previews daily and clean them up.
+name: Cleanup branch resources (Daily)
+on:
+  schedule:
+    - cron: '0 2 * * *' # Every day at 2 am
+jobs:
+  cleanup-previews:
+    name: Clean up dandling previews
+    runs-on: ubuntu-22.04
+    env:
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      AWS_DEFAULT_REGION: 'eu-west-1'
+    steps:
+      - name: Delete ${{ matrix.app }} preview app
```

Comment:
I think @mattrussell36 is right. We should install aws-cli first

https://github.com/marketplace/actions/install-aws-cli-action

Reply:
It's install by default on GH machines now.